app/controllers/publish/providers/accredited_partnerships_controller.rb

```diff
@@ -5,7 +5,13 @@ module Providers
     class AccreditedPartnershipsController < ApplicationController
       helper_method :accredited_provider_id
 
-      def index; end
+      def index
+        @provider_partnerships = @provider.accredited_partnerships
+      end
+
+      def show
+        @provider_partnership = provider.accredited_partnerships.find_by!(accredited_provider: partner)
```

Comment:
If you aren't using `@provider_partnership`, as in this case, it's ok to just set each provider to an instance variable. I know I told you different before, but this looks like a suitable exception.

```suggestion
        provider_partnership = provider.accredited_partnerships.find_by!(accredited_provider: partner)
        @accredited_provider = provider_partnership.accredited_provider
        @training_provider = provider_partnership.training_provider
```